XForm/XForm.Test/Functions/String/CoalesceFunctionTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.CodeAnalysis.Elfie.Model.Strings;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using XForm.Data;
+using XForm.Functions.String;
+
+namespace XForm.Test.Functions.String
+{
+    [TestClass]
+    public class CoalesceFunctionTests
+    {
+        [TestMethod]
+        public void Coalesce()
+        {
+            // Arrange
+
+            String8[] transformedArray = null;
+            bool[] nullArray = null;
+
+            // 1 a !   =   1
```

Comment:
It wasn't clear to me that Coalesce takes the first non-null/empty value. A comment here would be helpful.